data/collections/S1_RTC_seasonal_composite.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "s1-rtc-seasonal-composite",
+    "stac_version": "1.0.0",
+    "license": "not-provided",
+    "title": "Sentinel 1 RTC seasonal mosaic",
+    "type": "Collection",
+    "description": "Sentinel 1 RTC seasonal mosaic. The Sentinel-1 mission is a constellation of two polar-orbiting satellites, operating day and night performing C-band synthetic aperture radar imaging.",
+    "extent": {
+      "spatial": {
+        "bbox": [
+          [
+              -180, 50, 180, 77
```

Comment:
Did Nathan provide this? I ran a bbox over the files under the directory and got something a bit smaller

Reply:
The files currently there are just a sample. He said in the future this should be roughly the spatial extent. 

We can always correct these once we have to publish all the data.

Reply:
Curious -- how did you do that quickly ?